### ISSUE TITLE
Looks like Google Play Music is going away

### ISSUE THREAD

**Issue by tanimislam** [simon-weber/gmusicapi#673]
This issue is more like an announcement. [Google Play Music is shutting down](https://variety.com/2020/digital/news/google-play-music-shutdown-replaced-youtube-music-end-of-2020-1234724787/) by the end of 2020. Perhaps this project will go away unless we can repurpose it?

**Comment by simon-weber** [comment]
Yeah, I don't imagine gmusicapi will be very useful after the shutdown. I may archive this repo eventually; I'm not in any hurry to do anything about it.

**Comment by Crono141** [comment]
Is there an API for youtube music uploads?  If so, do you plan on creating a tool to automate uploads there?

**Comment by simon-weber** [comment]
[ytmusic](https://github.com/sigma67/ytmusicapi) claims to have upload support, but I haven't tried it myself. I don't have any plans to look into it myself as of now (I haven't decided how I'll replace my own usage of Google Music yet).

**Comment by tanimislam** [comment]
@simon-weber i would suggest Plex!

**Comment by ThomasRobertFr** [comment]
Hi, is there any chance that this wonderful api could be adapted to the replacement "youtube music"?
The UI of youtube music looks a lot like the UI of google music but I don't know how different their backend/api is similar. Considering they ask you to use a migration tool, they might be quite different sadly... 😞 

I use a lot the mopidy extension for gmusic that uses this library and am quite sad this might no longer work in a few months... 😞

**Comment by simon-weber** [comment]
Probably not; the systems are quite different behind the scenes.

**Comment by spidru** [comment]
Hi guys, I'm getting the error as shown in the screenshot. I guess this means GPM is truly dead?

![image](https://user-images.githubusercontent.com/11740200/94841022-b0182d00-0419-11eb-8e78-58a63481cfc1.png)

**Comment by simon-weber** [comment]
Could be; I don't think I've seen that one before.

I can still get into the webapp though.

**Comment by Emstar** [comment]
I was able to log in and fetch some music, but eventually downloads errored out. Trying to play music on the web client got me "Your music is temporarily unavailable from this device." Maybe I hit a cap.

**Comment by tjstum** [comment]
@simon-weber would you maybe like to disable issues or archive this repository? I'm bringing this acutely because of the random support request that landed in [#531](https://github.com/simon-weber/gmusicapi/issues/531#issuecomment-1018046002)

**Comment by simon-weber** [comment]
Sure, I'll archive it. Sorry about that.
